codegenerator/cli/templates/dynamic/codegen/src/TestHelpers.res.hbs

```diff
@@ -46,20 +46,21 @@ module EventFunctions = {
   The arguements that get passed to a "processEvent" helper function
   */
   @genType
-  type eventProcessorArgs<'eventArgs> = {
-    event: Types.eventLog<'eventArgs>,
+  type eventProcessorArgs<'event> = {
+    event: 'event,
```

Comment:
This is the only breaking change in public types. I don't think it's super critical, since it shouldn't be used standalone.